runtime/onert/core/src/ir/pass/ConstantInsertionPass.cc

```diff
@@ -57,15 +57,18 @@ void ConstantInsertionPass::callback(const OperationIndex &node_index, Operation
       // Update op_seq
       if (_lowered_graph.op_seqs().at(op_sequence_index).getInputs().contains(input))
       {
-        _lowered_graph.op_seqs().at(op_sequence_index).replaceInput(input, replaced_input);
+        // All inputs of op_seq have the same PermuteFactor because those inputs are inputs of first
+        // operation
+        _lowered_graph.op_seqs().at(op_sequence_index).replaceInputs(input, replaced_input);
       }
 
-      // Update current input operand only. Don't update all input node in this operation.
-      *it = replaced_input;
+      // Update the same inputs of a node at once because inputs of an operation have the same
+      // PermuteFactor
```

Comment:
@ragmani I believe your change would be right. However, I am not sure because I don't understand your assumption in your comment. At least for me, the previous code was intuitive and readable because it does not require the assumption.

Reply:
For me, Using set with unique inputs is more intutive and readable than using iterator. Changed code prevents duplicated callings and getting away from thinking why iterator replaces input one by one but `op_seqs` replaces at once.

Reply:
@ragmani I am not sure it is a bottleneck of duplicated calling. It does more by introducing set. It needs to create a set. Every insertion needs to check whether it has already the same element. I am not saying the changed code has overhead. Just I am not sure. I wonder what is the purpose of this PR. For efficiency or correctness? Code gets complicated at least for me.

Reply:
My purpose was for correctness. Moreover this change is more efficiency.
And I changed making unique `OperandSequence` from using `unordered_set` to use a method. Now changed code is simple and more intuitive.